src/graphql/queries.js

```diff
@@ -39,7 +39,13 @@ subscription OnWorkflowDeltasData ($workflowId: ID) {
 fragment WorkflowTreeDeltas on Deltas {
   id
   shutdown
-  added {
+  # NOTE: We set stripNull here so that values that are null in the data store
+  # are not stripped. Otherwise we could, for example, receive a task-proxy
+  # without the isHeld false/true entry. That happens when there is no
+  # value for isHeld, and then it is stripped. When stripNull: false is used,
+  # we will actually get the default value from Protobuf (in the case of
+  # isHeld, we will get isHeld: false, which is what we want for the UI).
+  added (stripNull: false) {
     ...WorkflowTreeAddedData
   }
   updated {
```

Comment:
Should we use `stripNull` here too?

Reply:
Good question. From what I understood, if we add `stripNull: false` (it is `true` at the moment as it's inheriting from an upper node), and there is an object in the deltas pool that does not contain the `isHeld`, for example, then that would mean getting a `isHeld: false`, since that is the default value from Protobuf for booleans.

That would mean that an object from the deltas pool without `isHeld` would cause the UI to show the task as not held... though that may not be what we wanted (perhaps a poor example, sorry!)

Reply:
Sorry, only just realised this is set to `stripNull: false` and remembered that `stripNull` cascades down from higher levels of the query.

Yes this makes sense. I don't think we should ever use `stripNull: true` on any of the `added` fields?

Reply:
Don't think so. Hopefully this change fixes our issue without breaking something that we are not able to foresee here :crossed_fingers: